simulator/Cargo.toml

```diff
@@ -25,3 +25,5 @@ serde_json = { workspace = true }
 tracing = { workspace = true }
 tracing-subscriber = { workspace = true }
 tikv-jemallocator = "0.6.0"
+reqwest = { version = "0.11", features = ["json", "blocking"] }
+tokio.workspace = true
```

Comment:
This has never worked for me for some reason, think I'd prefer if you kept as `{ workspace = true }`.

Plus I think reqwest is on the root `Cargo.toml`, you should be able to use it directly.

Reply:
I removed the tokio line since its not necessary, but I still get an error when reqwest is not defined in the simulator toml file. I don't see it defined in the root `Cargo.toml` file, and strangely it doesn't work when I add it either.
```rust
error[E0433]: failed to resolve: use of undeclared crate or module `reqwest`
 --> simulator/src/delegate.rs:5:5
  |
5 | use reqwest::blocking::Client;
  |     ^^^^^^^ use of undeclared crate or module `reqwest`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `mugraph-simulator` (lib) due to 1 previous error
```

Reply:
You need to move it to the root one, then use `reqwest = { workspace = true }` inside the crate.